package.json

```diff
@@ -77,4 +77,4 @@
     "_comment": "Replace 'reactTypescript' with 'react' if you want use React without Typescript",
     "projectType": "reactTypescript"
   }
```

Comment:
Don't add package.json to your pull request

Reply:
everyone keeps saying this to me😂